### DISCUSSION TITLE
Achievments!

### DISCUSSION THREAD

**Discussion by TobiasMelkersson** [drknzz/GitHub-Achievements#11] [Q&A] [answered]
New badge plz!

**Comment by drknzz** [comment]
You can answer other discussions I've created, so I can mark them as the answers.

**Comment by TiwariAbhishek23** [comment, marked as answer]
> New badge, please!

I also want to earn one. Let's help each other and make something great !!!

**Reply by drknzz** [reply]
Enjoy your badge my friend :)
